src/lib/helpers.js

```diff
@@ -114,6 +114,40 @@ export function createHelpers({ cache, webAPI }) {
       return data
     },
 
+    async externalRefs(extRef) {
```

Comment:
I would remove the async and use else/if to skip checking each case on each iteration.